Cmdline/Action/List.cs

```diff
@@ -111,13 +126,63 @@ public int RunCommand(CKAN.KSP ksp, object raw_options)
 
             if (!(options.porcelain) && exportFileType == null)
             {
-                user.RaiseMessage("\r\nLegend: -: Up to date. X: Incompatible. ^: Upgradable. ?: Unknown. *: Broken. ");
+                user.RaiseMessage("\r\nLegend: -: Up to date. X: Incompatible. ^: Upgradable. >: Replaceable\r\n        A: Autodetected. ?: Unknown. *: Broken. ");
                 // Broken mods are in a state that CKAN doesn't understand, and therefore can't handle automatically
             }
 
             return Exit.OK;
         }
 
+        private string ReplacmentModInfo (RelationshipDescriptor replacedBy, IRegistryQuerier registry, Versioning.KspVersionCriteria versionCriteria)
```

Comment:
Slight typo, ReplacmentModInfo vs ReplacementModInfo

Reply:
... but the logic of the function looks solid to me. :+1:

Reply:
Could this function call `IRegistryQuerier.GetReplacement`? They look similar. That way the logic would be even more centralized, and if we decide to make it more complex in the future we can just update that one Core function and `CmdLine.List` would get the changes for free.

Reply:
Yeah, I based GetReplacement on this. I will have a look to see if I can re-use that code reasonably. I don't see why not.

Reply:
And.... done!